### DISCUSSION TITLE
Unable to Access GitHub Account

### DISCUSSION THREAD

**Discussion by gopika0123** [community/community#114243] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello GitHub Community,

I hope you're doing well. I'm reaching out because I'm encountering an issue with accessing my GitHub account and would greatly appreciate any assistance or guidance.

 initially registered my GitHub account with my institutional email address. Later, I changed the primary email address associated with the account to my personal email. However, after some time, I found that I am unable to sign in to my account. When I attempt to log in, it seems to create a new GitHub account using the same email address.
Steps Taken:

Checked both my primary and institutional email inboxes for any verification or notification emails from GitHub.
Attempted to reset the password for my GitHub account.
Verified the change of my primary email address.
Checked for any potential suspension or restriction on my account.
Despite these efforts, I have been unable to regain access to my original GitHub account.

If anyone has experienced a similar issue or has any suggestions on how to resolve this matter, I would greatly appreciate your input.

Thank you in advance for your help!

Best regards,
Gopika

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @gopika0123! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

I've gone ahead and moved this to the correct category for you. Good luck!
